public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts

```diff
@@ -86,43 +88,44 @@ export class ConceptChartsComponent implements OnInit, OnDestroy {
 
   // Organize genders and set the chart title for the gender charts for simple display
   organizeGenders(analysis: Analysis) {
-    let otherCountValue = 0;
-    const others = [];
-
-    // No need to do anything if only one gender
-    if (analysis && analysis.results.length <= 1) {
-      return;
-    }
-    const results = [];
-    for (const g of analysis.results) {
-      if (g.stratum2 === this.dbc.MALE_GENDER_ID) {
-        this.maleGenderResult = g;
-        this.maleGenderChartTitle = g.analysisStratumName + ' - ' + g.countValue.toLocaleString();
-      } else if (g.stratum2 === this.dbc.FEMALE_GENDER_ID) {
-        this.femaleGenderResult = g;
-        this.femaleGenderChartTitle = g.analysisStratumName + ' - ' + g.countValue.toLocaleString();
-      } else {
-        otherCountValue += g.countValue;
+      // No need to do anything if only one gender
+      if (analysis && analysis.results.length <= 1) {
+          return;
```

Comment:
nit: rest of the file uses 2sp indentation - would match that throughout

Reply:
Done.